packages/aws-cdk-lib/aws-route53/test/record-set.test.ts

```diff
@@ -188,6 +188,36 @@ describe('record set', () => {
     });
   });
 
+  test('A record with health check', () => {
```

Comment:
I would also add a test to make sure that imported health checks can be properly mapped to Record sets